spec.txt

```diff
@@ -2376,7 +2367,7 @@ the first line meets both the [start condition] and the [end
 condition], the block will contain just that line.
 
 1.  **Start condition:**  line begins with the string `<script`,
-`<pre`, or `<style` (case-insensitive), followed by whitespace,
+`<pre`, or `<style` (case-insensitive), followed by a space, a tab,
```

Comment:
```suggestion
`<pre`, or `<style` (case-insensitive), followed by spaces or tabs,
```

Reply:
As this is about exactly one character, I’d say this is fine.